src/manual/multiple-peerconnections/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+  <title>WebRTC Test page</title>
+  <script type="text/javascript"
+src="../../../node_modules/webrtc-adapter/out/adapter.js"></script>
+  <script type="text/javascript">
+  function ignore1(err) {}
+  function VideoPair() {
+    this.localConnection = null;
+    this.remoteConnection = null;
+
+    var table = document.getElementById('test-table');
+    var row = table.insertRow(-1);
+
+    var localCell = row.insertCell(-1);
+    this.localView = document.createElement('video');
+    this.localView.autoplay = true;
+    localCell.appendChild(this.localView);
+
+    var remoteCell = row.insertCell(-1);
+    this.remoteView = document.createElement('video');
+    this.remoteView.autoplay = true;
+    remoteCell.appendChild(this.remoteView);
+
+    this.startTest = function() {
+      getUserMedia({audio: true, video: true},
+                   this.getUserMediaOkCallback.bind(this),
+                   this.getUserMediaFailureCallback.bind(this));
+    };
+
+    this.getUserMediaOkCallback = function(localStream) {
+      attachMediaStream(this.localView, localStream);
+      this.callUsingStream(localStream);
+    };
+
+    this.getUserMediaFailureCallback = function(error) {
+      console.error('getUserMedia request failed: name=' + error.name +
+                    ' constraintName=' + error.constraintName + ' message=' +
+                    error.message);
+    };
+
+    this.callUsingStream = function(localStream) {
+      this.localConnection = new RTCPeerConnection(null, null);
+      this.remoteConnection = new RTCPeerConnection(null, null);
+
+      this.localConnection.onicecandidate =
+        this.onIceCandidateToLocal.bind(this);
+      this.localConnection.addStream(localStream);
+
+      this.negotiate();
+    };
+
+    this.negotiate = function() {
+      this.localConnection.createOffer(
+          this.onOfferCreated.bind(this),
+          function(err) {});
+    };
+
+    this.onOfferCreated = function(offer) {
+      var self = this;
+      this.localConnection.setLocalDescription(offer, function() {
```

Comment:
Use promise instead of callback, see https://github.com/webrtc/samples/blob/gh-pages/src/content/peerconnection/pc1/js/main.js#L176